src/style.css

```diff
@@ -0,0 +1,48 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: lightgray;
+  color: rgb(44, 41, 41);
+  min-height: 100vh;
+}
+
+h2 {
+  min-height: 20vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.5rem;
+}
+
+input,
+button {
+  padding: 1rem;
+  font-size: 2rem;
+  border: 1px solid grey;
+  background: white;
+}
+
+button {
+  background: white;
+  color: white;
+}
+
+.removeBtn {
+  background: lightgray;
+  color: grey;
+}
+
+#container {
+  min-height: 20vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.5rem;
+  border: 1px solid grey;
+  margin: 50px;
+}
```

Comment:
- [ ] It would be better if the app looks the same as the given design. As one of the requirements. Kindly fix this minor issue. 

The given design 

![68747470733a2f2f696d672e796f75747562652e636f6d2f76692f416355642d5f596a6a71672f302e6a7067](https://user-images.githubusercontent.com/64173189/139961244-17cd3d33-56bf-4274-a51e-f70a75de1ec8.jpg)

Your design

![Capture](https://user-images.githubusercontent.com/64173189/139961283-df4d0c77-16d4-4d0e-801c-d4c757700d9b.JPG)